src/pages/SettingsPage.tsx

```diff
@@ -60,22 +62,80 @@ export default function SettingsPage() {
     useEffect(() => {
         const timer = setTimeout(updateCfgText, 0);
         return () => clearTimeout(timer);
-    }, [allProfiles, selectedProfile, keepLauncherOpen, developerMode])
+    }, [allProfiles, selectedProfile, keepLauncherOpen, developerMode, UITheme])
 
     return (
         <MainPanel>
-            <ToggleSection
-                text="Keep launcher open"
-                subtext="Prevents the launcher from closing after launching the game."
-                isChecked={keepLauncherOpen}
-                setIsChecked={setKeepLauncherOpen}
-            />
+
+            <div className="border-b-[2px] border-solid border-b-[#1E1E1F] p-[8px] bg-[#48494A]">
+                <div className="flex items-center justify-center">
+                    <div>
+                        <p className="minecraft-seven text-white text-[14px]">{"Keep launcher open"}</p>
+                        <p className="minecraft-seven text-[#BCBEC0] text-[12px]">{"Prevents the launcher from closing after launching the game."}</p>
+                    </div>
+                    <div className="ml-auto">
+                        <MinecraftToggle isChecked={keepLauncherOpen} setIsChecked={setKeepLauncherOpen}/>
+                    </div>
+                </div>
+            </div>
+
             <ToggleSection
                 text="Developer mode"
                 subtext="Enables hot-reloading and prompting to attach a debugger."
                 isChecked={developerMode}
                 setIsChecked={setDeveloperMode}
             />
+
+            <div className="border-y-[2px] border-solid border-b-[#1E1E1F] border-t-[#5A5B5C] p-[8px] bg-[#48494A]">
```

Comment:
This entire section for the radial buttons should be made generic and placed into its own component

Reply:
done